backend/dal/Helpers/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -1,40 +0,0 @@
-using Microsoft.AspNetCore.Builder;
-using Microsoft.EntityFrameworkCore;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Logging;
-using System;
-using System.Diagnostics.CodeAnalysis;
-
-namespace Pims.Dal.Helpers.Extensions
-{
-    /// <summary>
-    /// ApplicationBuilderExtensions static class, provides extension methods for ApplicationBuilder objects.
-    /// </summary>
-    ///
-    [ExcludeFromCodeCoverage]
-    public static class ApplicationBuilderExtensions
-    {
-        /// <summary>
-        /// Initialize the database when the application starts.
-        /// This isn't an ideal way to do this, but will work for our purposes.
-        /// </summary>
-        /// <param name="app"></param>
-        public static void UpdateDatabase<T>(this IApplicationBuilder app)
-        {
-            using var serviceScope = app.ApplicationServices
-                .GetRequiredService<IServiceScopeFactory>()
-                .CreateScope();
-            var logger = serviceScope.ServiceProvider.GetService<ILogger<T>>();
-
-            try
-            {
-                using var context = serviceScope.ServiceProvider.GetService<PimsContext>();
-                context.Database.Migrate();
```

Comment:
The purpose of this class is to run migrations on app startup, no longer needed.